### PR TITLE
updates for prod json

### PR DESCRIPTION
**Pull request by mozrokafor**
updating json for release 2.12

### CODE REVIEW

File: docs/mozillavpn.json [hunk 1]
```diff
@@ -111,8 +111,8 @@
     },
     "2.12.0": {
       "build": null,
-      "date": "2022-12-15",
-      "platforms": ["MACOS", "WINDOWS"]
+      "date": "2022-12-16",
```
Comment (bakulf) [inline docs/mozillavpn.json:114]: do not change the date.

**Review by bakulf — CHANGES_REQUESTED** [review]
r+wc

File: docs/mozillavpn.json (hunk 1 above)
Comment (mozrokafor) [reply docs/mozillavpn.json]: @birdsarah  you happen to know what this date is for?

**Review by birdsarah — APPROVED** [review]

File: docs/mozillavpn.json (hunk 1 above)
Comment (birdsarah) [reply docs/mozillavpn.json]: it's just for record keeping it has no functional impact. it should reflect the actual day of release but given that there's multiple release dates and we don't have a feature for that we just pick "the best one" which I'd say is usually when Mac and Windows go out.

**Pull request merged**